core/parentchain/indirect-calls-executor/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "itc-parentchain-indirect-calls-executor"
+version = "0.8.0"
+authors = ["Integritee AG <hello@integritee.network>"]
+edition = "2018"
+resolver = "2"
+
+[dependencies]
+# sgx dependencies
+sgx_types = { branch = "v1.1.4-testing", git = "https://github.com/apache/teaclave-sgx-sdk.git" }
+sgx_tstd = { branch = "v1.1.4-testing", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
+
+# local dependencies
+ita-stf = { path = "../../../app-libs/stf", default-features = false }
+itp-extrinsics-factory = { path = "../../../core-primitives/extrinsics-factory", default-features = false }
+itp-ocall-api = { path = "../../../core-primitives/ocall-api", default-features = false }
+itp-settings = { path = "../../../core-primitives/settings" }
+itp-sgx-crypto = { path = "../../../core-primitives/sgx/crypto", default-features = false }
+itp-stf-executor = { path = "../../../core-primitives/stf-executor", default-features = false }
+itp-types = { path = "../../../core-primitives/types", default-features = false }
+
+# sgx enabled external libraries
+rust-base58_sgx = { package = "rust-base58", rev = "sgx_1.1.3", git = "https://github.com/mesalock-linux/rust-base58-sgx", optional = true, default-features = false, features = ["mesalock_sgx"] }
+thiserror_sgx = { package = "thiserror", git = "https://github.com/mesalock-linux/thiserror-sgx", tag = "sgx_1.1.3", optional = true }
+
+# std compatible external libraries (make sure these versions match with the sgx-enabled ones above)
+thiserror = { version = "1.0", optional = true }
+rust-base58 = { package = "rust-base58", version = "0.0.4", optional = true }
```

Comment:
It might make sense to replace this lib with a no std compatible one:

https://github.com/encointer/pallets/blob/33cc0a965c29ab21ab0c68f1fb297a9359dfbfaf/primitives/Cargo.toml#L8

Reply:
hm yes, that would save some trouble in the toml and the code - I'll try to make the change 👍